go/vt/wrangler/reparent.go

```diff
@@ -125,9 +125,9 @@ func (wr *Wrangler) InitShardMaster(ctx context.Context, keyspace, shard string,
 		Force:                   force,
 	}, waitReplicasTimeout, wr.tmc, wr.logger)
 	if err != nil {
-		event.DispatchUpdate(ev, "failed InitShardMaster: "+err.Error())
+		event.DispatchUpdate(ev, "failed InitShardPrimary: "+err.Error())
 	} else {
-		event.DispatchUpdate(ev, "finished InitShardMaster")
+		event.DispatchUpdate(ev, "finished InitShardPrimary")
```

Comment:
What do you think of this? The event will be `InitShardPrimary` even if the command used by the user is `InitShardMaster`.

Reply:
Changing things now seems good to me. I don't see anything in the code or docs that indicates we rely on (or can expect to rely) the exact update message text.